### DISCUSSION TITLE
"ZEPETO.World .unitypackage" 월트템플릿을 유니티에 임포트했는데 에러메시지가 떠요.

### DISCUSSION THREAD

**Discussion by heodoly8672** [naverz/zepeto-studio-kor#926] [5. Q&A]
안녕하세요.
이제 막 월드 만들어보려는 제페토월드 초보입니다.
월드 템플릿을 유니티내에 임포트해서 등록했는데...에러메시지가 뜨는데 큰 문제 일까요?

맨밑 하단 메시지 ->ExcutionEngineException: Srting conversion error: lllegal byre seequence encounted in the input.

console탭 메시지 -.> [18:12:23] ExcutionEngineException: Srting conversion error: lllegal byre seequence encounted in the input.
                                   System.Reflection.Assembly.GetCodeBase (System.Boolean escaped) (at <695d1cc93cca45069c528c15c9fdd749>:0)

이라는 에러메시지가 뜹니다.
![유니티오류](https://user-images.githubusercontent.com/108266982/175908420-684d74ce-9db2-4162-b4b6-b1810154f502.jpg)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 최근 윈도우 PC에서 제페토 템플릿을 import할때 동일한 오류를 겪는 분들이 많으신 것 같습니다.
사용하신 개발 환경 (윈도우 버전)을 알려주시면 문제 해결이 도움이 될 것 같습니다.
새로운 프로젝트를 생성하신 후 다시 import 해도 안되는 걸까요?
